src/oemof/solph/_options.py

```diff
@@ -247,6 +247,8 @@ def __init__(
         for attribute, value in custom_attributes.items():
             setattr(self, attribute, value)
 
+        self.first_flexible_timestep = max(minimum_uptime, minimum_downtime)
```

Comment:
Understanding you correct: The first timestep is forced to be the maximum. should it not be deepding on the initial_status?

Reply:
No. This is a direct translation from the old `NonConvex.max_up_down`: The first time step that allows flexibility is after the maximum of minimum_uptime and minimum_downtime. This does not make sense, though, as minimum_uptime should be relevant if the initial status is on, and minimum_downtime if the initial status is off.